cmd/juju/controller/addmodel.go

```diff
@@ -396,6 +398,16 @@ there is no default cloud defined, please specify one using:
 	return cloudTag, cloud, nil
 }
 
+var ambiguousDetectedCredentialError = errors.New(`
+more than one credential detected
+run juju autoload-credentials and specify a credential using the --credential argument`[1:],
+)
+
+var ambiguousCredentialError = errors.New(`
+more than one credential is available
```

Comment:
Similar to above comment, could we tell the user to run "juju credentials" to see what credentials there are to choose from.